systems/plants/RigidBodyTorsionalSpring.m

```diff
@@ -14,7 +14,7 @@
     function [f_ext,df_ext] = computeSpatialForce(obj,manip,q,qd)
       theta = q(manip.body(obj.child_body).position_num);
       torque = obj.k*(obj.rest_angle - theta);
-      f_ext = sparse(6,getNumBodies(manip));
+      f_ext = q(1)*sparse(6,getNumBodies(manip));
```

Comment:
same problem in line 348 of RigidBodyManipulator.m:

damping = zeros(nv, 1);

should be changed to:
       
damping = v(1)*zeros(nv, 1);